assets/yarn.lock

```diff
@@ -4300,6 +4339,11 @@ signal-exit@^3.0.0:
   resolved "https://registry.yarnpkg.com/signal-exit/-/signal-exit-3.0.2.tgz#b5fdc08f1287ea1178628e415e25132b73646c6d"
   integrity sha1-tf3AjxKH6hF4Yo5BXiUTK3NkbG0=
 
+signal-exit@^3.0.2:
```

Comment:
I wonder why this and the previous resolution aren't resolved by the same version

Reply:
I forced this to resolve to the same version by deleting these two entries and running yarn install again